datanode/src/main/java/com/odysseusinc/arachne/datanode/controller/AuthController.java

```diff
@@ -59,67 +49,43 @@ public class AuthController {
     private UserService userService;
 
     @Autowired
-    private Authenticator authenticator;
-
-    @Autowired
-    private UserRegistrationStrategy userRegisterStrategy;
+    private CredentialsService credentialsService;
 
     @Autowired
-    private AccessTokenResolver accessTokenResolver;
-
-    @Value("${security.method}")
-    private String authMethod;
-
-    @Value("${security.authentication.mode:" + AuthenticationMode.Const.STANDARD + "}")
-    private AuthenticationMode authenticationMode = AuthenticationMode.STANDARD;
-
+    private DbBasicCredentialsService basicCredentialsService;
 
     @ApiOperation("Get authentication mode")
     @RequestMapping(value = "/api/v1/auth/mode", method = GET)
     @Deprecated
     public CommonAuthenticationModeDTO authenticationMode() {
```

Comment:
Where are we even using this enpoint?

Reply:
I'm not sure. I don't even think that method is used anywhere. I kept it just in case; I'm going to handle that with the IAP implementation.